Extractor/UAExtractor.cs

```diff
@@ -959,6 +959,14 @@ private async Task PushNodes(PusherInput input)
 
             bool initial = input.Variables.Count() + input.Objects.Count() >= State.NumActiveNodes;
 
+            if (initial && input.Variables.Any() && Config.Subscriptions.DataPoints)
```

Comment:
The only way for this if to evaluate true is for there to be any input variables, but the message is saying there are no variables. Is this check correct?

Reply:
Oops, yeah, it should be the inverse.